src/balena.sbvr

```diff
@@ -516,6 +516,7 @@ Fact type: release has source
 	Necessity: each release has exactly one source.
 Fact type: release has build log
 	Necessity: each release has at most one build log.
+Fact type: release is invalidated
```

Comment:
We are open to other names if you have any better suggestions, this is just what we came up with at the arch call.

Reply:
`is obsolete`?

Reply:
@dfunckt to me `obsolete` implies that it's out of date, but it's still fine to use it, which is not the case here, not sure if it's just me though :)

Reply:
Yeah, 'obsolete' means 'out of date but still usable', 'invalidated' means 'rejected due to error'.

Reply:
Okay gotcha -- I agree `is obsolete` isn't at all correct. I was more concerned with "is invalidated" meaning it's something it can be replaced with a new version (ie. "cache invalidation"), which in the case of a single release doesn't make sense to me, but if seen as a whole might :)